docs/content/en/contribute/general/github/_index.md

```diff
@@ -0,0 +1,274 @@
+---
+title: Working with github
+description: Information relevant to github
+weight: 100
+---
+
+To start your contribution with [Keptn Lifecycle Toolkit](https://github.com/keptn/lifecycle-toolkit) follow this github instructions:
+
+### Fork and clone the repository
```

Comment:
Please be consistent with heading levels.

The title in the frontmatter works as the heading of the first level or markdown `# heading 1` - but there is no heading of the second level, just the third level. Please adapt to have a consistent heading-level hierarchy. Thank you

Reply:
@aepfli sorry for the late response you can review now. :)